### ISSUE TITLE
*MODDED* Most Vive controller controls not working (Movement/Mining/Building)

### ISSUE THREAD

**Issue by TheKatsar** [jrbudda/minecrift#330]
I'm trying to run vivecraft on the Augmented Triad modpack v0.9.5.

VC installed fine and the game launches and runs smooth, however once I load into a world I am unable to locomote (IRL movement tracks fine), mine/attack or build/interact, the buttons on the controllers simply dont work. 

This issue does not persist in menus, leading me to believe that it is some sort of keymapping issue or, more likely, a conflict with a movement mod. My install with "vanilla" minecraft (VC only) works fine, so it isnt an issue with an improper install or faulty hardware.

I have tried disabling a few movement related mods, but with the size of the pack I cant remember them all. Any help or advice given is appreciated

-Katsar.

(Not that it should make much difference but just in case my specs are:
Asus Z-170A MoBo
Zotac GTX-1080 GPU
16gb Corsair Dominator Platinum 3200 RAM
i7-6700k CPU @ 4.0Ghz
Windows 8.1 x64 OS)

**Comment by jrbudda** [comment]
NotEnoughKeys in the pack?

**Comment by TheKatsar** [comment]
Yes, it was, I disabled it and now it works perfectly. I guess I overlooked it when checking the compatibility list against the pack. Thanks for the help man
